packages/icons/src/index.d.ts

```diff
@@ -80,14 +80,20 @@ export { default as OneFilled } from './components/oneFilled';
 export { default as List } from './components/list';
 export { default as Seat } from './components/seat';
 export { default as FrontCar } from './components/frontCar';
-export { default as Update } from './components/update';
+export { default as Function } from './components/function';
+export { default as Phone } from './components/phone';
+export { default as Envelope } from './components/envelope';
+export { default as InteractiveExamples } from './components/interactiveExamples';
+export { default as YoutubeIcon } from './components/youtubeIcon';
+export { default as MediumIcon } from './components/mediumIcon';
```

Comment:
I think we should export this icons as Youtube and Medium without the Icon suffix